### PR TITLE
Fixing broken link to Jenkinsfile

### PR DESCRIPTION
**Pull request by marjanstankovic**
The correct path to the Jenkinsfile is ./integrations/jenkins/Jenkinsfile

Signed-off-by: Marjan Stankovic <marjan.stankovic@gmail.com>

Please ensure that your pull request adheres to our [contribution guidelines](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md). Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,8 +149,8 @@ or
 ## Running on CI
 
 A basic ORT pipeline (using the _analyzer_, _scanner_ and _reporter_) can easily be run on
-[Jenkins CI](https://jenkins.io/) by using the [Jenkinsfile](./integrations/Jenkinsfile) in a (declarative)
-[pipeline](https://jenkins.io/doc/book/pipeline/) job. Please see the [Jenkinsfile](./integrations/Jenkinsfile) itself
+[Jenkins CI](https://jenkins.io/) by using the [Jenkinsfile](./integrations/jenkins/Jenkinsfile) in a (declarative)
+[pipeline](https://jenkins.io/doc/book/pipeline/) job. Please see the [Jenkinsfile](./integrations/jenkins/Jenkinsfile) itself
```
Comment (sschuberth) [inline README.md:153]: Would you mind changing the second sentence to instead link to the new dedicated `README.md` which documents the required Jenkins plugins?

**Review by sschuberth — CHANGES_REQUESTED** [review]
Thanks for your contribution!

File: README.md (hunk 1 above)
Comment (marjanstankovic) [reply README.md]: Done

**Comment by sschuberth** [comment]
Just some general remark about your commits: I realized you use "marjanstankovic" as the author name / for the signoff. Shouldn't that rather be "Marjan Stankovic"?

**Comment by marjanstankovic** [comment]
> Just some general remark about your commits: I realized you use "marjanstankovic" as the author name / for the signoff. Shouldn't that rather be "Marjan Stankovic"?

I changed it above in the comment. When doing the sign-off, I simply left the auto-generated text. Now I don't know how to change it now for the commits. Next time I will take care  :)

**Comment by marjanstankovic** [comment]
> Just some general remark about your commits: I realized you use "marjanstankovic" as the author name / for the signoff. Shouldn't that rather be "Marjan Stankovic"?

I changed it above in the comment. When doing the sign-off, I simply left the auto-generated text. Now I don't know how to change it now for the commits. Next time I will take care  :)

**Comment by sschuberth** [comment]
> Now I don't know how to change it now for the commits. Next time I will take care :)

As we are a bit picky about the Git history in this project, I'd like to kindly ask you to already fix it for this PR by rewriting your commits and force-pushing to this PR's branch. See it as a Git exercise 😉 You may want to follow [these instructions](https://stackoverflow.com/a/3042512/1127485).

**Pull request closed without merging**